src/main/java/org/ethereum/beacon/discovery/DiscoveryManager.java

```diff
@@ -46,4 +47,6 @@
 
   /** Sends the TALKREQ so the specified {@code node} and returns the TALKRESP promise */
   CompletableFuture<Bytes> talk(NodeRecord nodeRecord, Bytes protocol, Bytes request);
+
+  Stream<NodeRecord> streamActiveSessions();
```

Comment:
should be `@MustBeClosed`?

Reply:
No need for that - it isn't from any resource that needs to be closed.